lib/v2/oreno3d/main.js

```diff
@@ -14,81 +14,124 @@ const sortRename = {
-    const $ = cheerio.load(response.data);
-    // 第一页，获取搜索主标题
-    const title = $('div.g-main-list').find('h1.main-h').text(); // 搜索标题;//第二页跳转链接
-
-    const list = $(selector);
-    const items = await Promise.all(
-        list
-            .map((_, item) => {
+    // 数据爬取和分析
+    async function getData(userUrl) {
+        const selector = 'a.box';
+        const response = await got(userUrl);
+        const $ = cheerio.load(response.data);
+        // 第一页，获取搜索主标题
+        const title = $("div.g-main-list").find("h1.main-h").text();// 搜索标题
+        const list = $(selector);
+        const items = await Promise.all(
+            list.map(async(_, item) => {
                 // 第一页，获取搜各视频地址
                 const link = $(item).attr('href');
-                return ctx.cache.tryGet(link, async () => {
-                    // 第二页数据分析
-                    const sec_data = await get_sec_page_data(link);
-                    // 传递
-                    const raw_pic_link = sec_data.raw_pic_link;
-                    const video_name = sec_data.video_name;
-                    const authors = sec_data.authors;
-                    const origins = sec_data.origins;
-                    const characters = sec_data.characters;
-                    const tags = sec_data.tags;
-                    const desc = sec_data.desc;
-                    const iwara_link = sec_data.iwara_link;
-                    const oreno3d_link = sec_data.oreno3d_link;
-                    // 打包
-                    return {
-                        title: `${video_name} - ${authors}`,
-                        author: authors,
+                // 第二页数据分析+缓存
+                const sec_data = await get_sec_page_data(link);
+                // 传递
+                const raw_pic_link = sec_data.raw_pic_link;
+                const video_name = sec_data.video_name;
+                const authors = sec_data.authors;
+                const origins = sec_data.origins;
+                const characters = sec_data.characters;
+                const tags = sec_data.tags;
+                const desc = sec_data.desc;
+                const iwara_link = sec_data.iwara_link;
+                const oreno3d_link = sec_data.oreno3d_link;
+                // 打包,缓存HTML
+                const description = await ctx.cache.tryGet(iwara_link, async () => {
+                    const result = await art(path.join(__dirname, 'templates/description.art'), {
+                        raw_pic_link,
+                        video_name,
+                        authors,
+                        origins,
+                        characters,
+                        tags,
+                        desc,
+                        iwara_link,
+                        oreno3d_link
+                    });
+                    return result;});
+                const title = `${video_name} - ${authors}`;
+                const realData = await ctx.cache.tryGet(title, () => {
```

Comment:
```suggestion
                const realData = await ctx.cache.tryGet(oreno3d_link, () => {
```
Please use the site link as the cache key